salt/modules/status.py

```diff
@@ -1025,419 +1025,16 @@ def generic_vmstats():
     return get_version.get(__grains__['kernel'], lambda: errmsg)()
 
 
-def nproc():
```

Comment:
Did you mean to remove all of these functions?

Reply:
Not quite :-) I've fixed that.